docs/README.md

```diff
@@ -753,16 +754,138 @@ public static async handle(event: StockMoved, register: Register): Promise<void>
 
 ### 4. Entities and reducers
 
+The source of truth of your Booster app are the events, but events make sense in the context of a domain entity.
+For example, in a banking app, there might be two events: `MoneyDeposited` and `MoneyWithdrawn`. But these events
+only make sense in the context of a `BankAccount`.
+
+Entities are created on the fly, by _reducing_ the whole event stream. They aren't saved anywhere, although Booster
+does create automatic snapshots in order to make the reduction process efficient. You are the one that defines the
+reducer function.
+
+An entity is defined as a class with the `@Entity` decorator. Inside of it there is a static method with
+the `@Reduces` decorator that also specifies the event the reducer is subscribed to. In Booster an entity looks like this:
+
+```typescript
+@Entity
+export class EntityName {
+  public constructor(readonly fieldA: SomeType, readonly fieldB: SomeOtherType /* as many fields as needed */) {}
+
+  @Reduces(SomeEvent)
+  public static reduceSomeEvent(event: SomeEvent, previousState?: EntityName): EntityName {
+    /* Return a new entity based on the previous one */
+  }
+}
+```
+
+Each time an event is generated, all reducers that are listening to it are triggered. Note that event ordering is
+preserved per entity instance. This means that if two events arrive at the same time at a reducer, the first one to
+be picked will be the one that was generated first.
```

Comment:
```suggestion
```

Reply:
I don't understand why removing this part of the sentence

Reply:
For me, the sentence is redundant after reading the paragraph, and it sounds like a sentence from the Master Yoda 😂  I'd also remove it.

Reply:
Because it's not needed after rephrasing the previous line